pkg/apis/core/types.go

```diff
@@ -5351,3 +5363,30 @@ type TopologySpreadConstraint struct {
 	// +optional
 	LabelSelector *metav1.LabelSelector
 }
+
+// PortStatus represents the condition of a service port condition
+type PortStatus struct {
+	Port     int32
+	Protocol Protocol
+	Ready    bool
```

Comment:
What does this mean?  Does it mean traffic is flowing or that it has been configured already or just that it is accepted and being configured?  Is "ready" the best name?

Reply:
Yeah, so from the KEP.

> "ready": {
      "description": "Specifies whether the port was configured for the load-balancer.",
      "type": "boolean"

Reply:
What about "configured" or maybe flip it and make it `error string` which references a condition name ("" means no error)